lib/membrane_rtc_engine/endpoints/webrtc/rtp_connection_allocator.ex

```diff
@@ -229,6 +250,11 @@ defmodule Membrane.RTC.Engine.Endpoint.WebRTC.RTPConnectionAllocator do
     {:noreply, state}
   end
 
+  @impl true
+  def handle_info({_pid, {:decrease_allocation_request, _response}}, state) do
+    {:noreply, state}
+  end
```

Comment:
Thought this can't come to us asynchronously, can it?

Reply:
We only wait for 10ms. This might actually need some tuning 🤔 .

Now that you made that comment, we probably should put a `warn` or something in here. We don't want that to happen too much, but it is an expected situation.